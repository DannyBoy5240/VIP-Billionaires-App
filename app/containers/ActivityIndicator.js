import React from 'react'
import { ActivityIndicator, StyleSheet } from 'react-native'
import { themes } from '../constants/colors'

const styles = StyleSheet.create({
  indicator: {
    padding: 16,
    flex: 1,
  },
  absolute: {
    position: 'absolute',
    left: 0,
    right: 0,
    top: 0,
    bottom: 0,
    alignItems: 'center',
    justifyContent: 'center',
    zIndex: 100000,
  },
})

const RCActivityIndicator = ({ theme, absolute, ...props }) => (
  <ActivityIndicator
    style={[styles.indicator, absolute && styles.absolute]}
    color={themes[theme].activeTintColor}
    {...props}
  />
)
export default RCActivityIndicator
