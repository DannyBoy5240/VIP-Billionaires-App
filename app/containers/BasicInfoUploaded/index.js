import React from 'react'
import { View, Text, Image } from 'react-native'

import styles from './style'
import { VectorIcon } from '../VectorIcon'
import images from '../../assets/images'
import { themes } from '../../constants/colors'
import { useTheme } from '../../theme'

const BasicInfoUploaded = ({ name, gender, dob, phone, location, showCloseIcon }) => {

  const { theme } = useTheme()

  return (
    <View style={styles.container}>
      <View>
        <Text style={[styles.name, { color: themes[theme].titleColor }]}>{name}</Text>
        <Text style={[styles.genderAndDob, { color: themes[theme].titleColor }]}>
          {gender} | {dob}
        </Text>
        <View style={styles.iconAndText}>
          <VectorIcon
            type="MaterialIcons"
            name="perm-phone-msg"
            color="#F5BF4D"
            size={18}
          />
          <Text style={[styles.phone, { color: themes[theme].titleColor }]}>{phone}</Text>
        </View>
        <View style={styles.iconAndText}>
          <Image
            source={images.location_home}
            style={styles.locationHomeImage}
          />
          <Text style={[styles.location, { color: themes[theme].titleColor }]}>{location}</Text>
        </View>
      </View>
      {showCloseIcon && (
        <VectorIcon
          type="Ionicons"
          name="close-outline"
          size={20}
          color="#858585"
          style={styles.closeIcon}
        />
      )}
    </View>
  );
}

export default BasicInfoUploaded