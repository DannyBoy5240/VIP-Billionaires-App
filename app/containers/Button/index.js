import React from 'react'
import { StyleSheet, View, Text, Image, ActivityIndicator, TouchableOpacity } from 'react-native'
import { RectButton } from 'react-native-gesture-handler'
import { COLOR_BLACK, COLOR_BTN_BACKGROUND, COLOR_BTN_BORDER, COLOR_WHITE, themes } from '../../constants/colors'
import images from '../../assets/images'
import { VectorIcon } from '../VectorIcon'

/* eslint-disable react-native/no-unused-styles */
const styles = StyleSheet.create({
  container: {
    borderRadius: 13,
    backgroundColor: COLOR_WHITE,
  },
  oauthButtonContainer: {
    margin: 8,
  },
  oauthBtn: {
    height: 40,
    resizeMode: 'contain',
  },
  buttonContainer: {
    borderRadius: 7,
    height: 45,
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center',
    paddingHorizontal: 16,
    borderWidth: 0.25,
  },
  icon: {
    position: 'absolute',
    left: 7.5,
    top: 7.5,
    width: 35,
    height: 35,
    resizeMode: 'contain',
  },
  text: {
    fontSize: 16,
    textAlign: 'center',
    fontFamily: 'Raleway',
    lineHeight: 18,
  },

  button_size_Z: {
    width: 335,
  },
  button_size_Y: {
    width: 201,
  },
  button_size_X: {
    width: 125,
  },
  button_size_W: {
    width: '100%',
  },
  button_size_V: {
    width: 125,
  },
  button_size_U: {
    width: 140,
  },
  button_size_T: {
    width: 313,
  },
  button_size_S: {
    width: 345,
  },
})

const Button = props => {
  const {
    title,
    text,
    icon,
    type,
    size,
    onPress,
    iconCenter,
    hidden,
    disabled,
    width,
    height,
    backgroundColor,
    loading,
    style,
    textColor: color,
    fontWeight,
    theme,
    iconColor,
    ...otherProps
  } = props

  if (hidden) {
    return null
  }

  if (type === 'oauth') {
    return (
      <RectButton onPress={onPress} style={styles.oauthButtonContainer}>
        <Image source={images[title]} style={styles.oauthBtn} />
      </RectButton>
    )
  }

  if (type === 'secondary') {
    return (
      <TouchableOpacity
        onPress={onPress}
        disabled={(disabled || loading)}
        style={[
          styles.container,
          size ? styles[`button_size_${size}`] : {},
          style,
        ]}
        {...otherProps}>
        <View
          style={[
            styles.buttonContainer,
            size ? styles[`button_size_${size}`] : {},
            width ? { width } : {},
            height ? { height } : {},
            {
              opacity: disabled ? 0.5 : 1,
              borderWidth: 0.5,
              backgroundColor: COLOR_WHITE,
              borderColor: COLOR_BTN_BORDER,
            },
          ]}>
          {icon ? (
            <VectorIcon
              type="MaterialCommunityIcons" name={icon}
              size={24} style={{ marginRight: 8, ...iconColor }} />
          ) : null}
          {loading ? (
            <ActivityIndicator color={COLOR_BLACK} />
          ) : (
            <>
              <Text style={[styles.text, { color: disabled ? themes[theme].textColor : COLOR_BLACK }]}>
                {text || title}
              </Text>
            </>
          )}
        </View>
      </TouchableOpacity>
    )
  }

  return (
    <TouchableOpacity
      onPress={onPress}
      disabled={(disabled || loading)}
      style={[
        styles.container,
        size ? styles[`button_size_${size}`] : {},
        style,
      ]}
      {...otherProps}>
      <View
        style={[
          styles.buttonContainer,
          size ? styles[`button_size_${size}`] : {},
          width ? { width } : {},
          height ? { height } : {},
          {
            backgroundColor: COLOR_BTN_BACKGROUND,
            borderColor: COLOR_BTN_BORDER,
          },
        ]}>
        {icon ? (
          <VectorIcon
            type="MaterialCommunityIcons" name={icon}
            size={24} style={{ marginRight: 8, ...iconColor }} />
        ) : null}
        {loading ? (
          <ActivityIndicator color={COLOR_WHITE} />
        ) : (
          <>
            <Text style={[styles.text, { color: disabled ? themes[theme].textColor : COLOR_WHITE }]}>
              {text || title}
            </Text>
          </>
        )}
      </View>
    </TouchableOpacity>
  )
}

Button.defaultProps = {
  title: 'Press me!',
  type: 'primary',
  size: 'Z',
  onPress: () => alert('It works!'),
  icon: null,
  hidden: false,
  disabled: false,
  loading: false,
}

export default Button
