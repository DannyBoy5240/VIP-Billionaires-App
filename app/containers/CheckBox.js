import React from 'react'
import { Text, TouchableOpacity } from 'react-native'
import Ionicons from 'react-native-vector-icons/Ionicons'
import { COLOR_BLACK_LIGHT, COLOR_BLUE } from '../constants/colors'

const CheckBox = React.memo(
  ({
     title,
     checked,
     checkedColor,
     unCheckedColor,
     checkedIcon,
     uncheckedIcon,
     textStyle,
     containerStyle,
     onPress,
   }) => (
    <TouchableOpacity
      style={{
        ...containerStyle,
        flexDirection: 'row',
        alignItems: 'center',
        height: 49,
        paddingHorizontal: 12,
        borderWidth: 1,
        borderRadius: 11,
        borderColor: checked ? COLOR_BLUE : COLOR_BLACK_LIGHT,
        marginBottom: 5,
      }}
      onPress={onPress}>
      <Ionicons
        name={checked ? checkedIcon : uncheckedIcon}
        size={12}
        color={checked ? checkedColor : unCheckedColor}
      />
      {title ? (
        <Text style={{ marginLeft: 8, ...textStyle }}>{title}</Text>
      ) : null}
    </TouchableOpacity>
  ),
)
export default CheckBox
