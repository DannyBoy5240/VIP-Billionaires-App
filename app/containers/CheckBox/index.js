import React from 'react'
import { Text, TouchableOpacity } from 'react-native'
import { COLOR_YELLOW } from '../../constants/colors'
import { VectorIcon } from '../VectorIcon'

const Index = React.memo(
  ({ title, checked, textStyle, containerStyle, onPress }) => (
    <TouchableOpacity
      style={{
        ...containerStyle,
        flexDirection: 'row',
        alignItems: 'center',
        paddingVertical: 8,
      }}
      onPress={onPress}>
      <VectorIcon
        name={checked ? 'check-square-o' : 'square-o'}
        type={'FontAwesome'}
        size={24}
        color={checked ? COLOR_YELLOW : 'black'}
      />
      {title ? (
        <Text style={{ marginLeft: 8, ...textStyle }}>{title}</Text>
      ) : null}
    </TouchableOpacity>
  ),
)

export default Index
