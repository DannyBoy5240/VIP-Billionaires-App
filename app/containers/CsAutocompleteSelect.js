import React, {useState} from 'react';
import {Dimensions, StyleSheet, View, Text} from 'react-native';
import {AutocompleteDropdown} from 'react-native-autocomplete-dropdown';
import {themes} from '../constants/colors';
import {VectorIcon} from './VectorIcon';

const {width, height} = Dimensions.get('window');

const styles = StyleSheet.create({
  container: {
    width: width * 0.92,
    borderWidth: 1,
    flexDirection: 'row',
    alignItems: 'center',
    alignSelf: 'center',
    paddingLeft: 10,
    borderRadius: 8,
    paddingHorizontal: 10,
    justifyContent: 'space-between',
    height: 56,
    borderColor: '#C4C4C4',
    marginBottom: 15,
  },
  label: {
    fontFamily: 'Raleway',
    fontWeight: '400',
    color: '#C4C4C4',
    marginBottom: 5,
    fontSize: 14,
  },
});

const CsAutocompleteSelect = ({
  leftIcon,
  data,
  onSelectItem,
  placeholder,
  theme,
  label,
}) => {
  const [show, setShow] = useState(false);

  const RightIcon = () => {
    return (
      <VectorIcon
        type={'Entypo'}
        name={show ? 'chevron-thin-up' : 'chevron-thin-right'}
        color={themes[theme].activeTintColor}
        size={18}
      />
    );
  };

  return (
    <>
      {label && (
        <Text style={[styles.label, themes[theme].normalTextColor]}>
          {label}
        </Text>
      )}
      <View style={styles.container}>
        <AutocompleteDropdown
          clearOnFocus={false}
          closeOnBlur={true}
          closeOnSubmit={false}
          dataSet={data}
          onSelectItem={onSelectItem}
          textInputProps={{
            placeholder: placeholder,
            autoCorrect: false,
            autoCapitalize: 'none',
            style: {
              fontFamily: 'Raleway',
              color: themes[theme].activeTintColor,
              fontSize: 14,
            },
          }}
          inputContainerStyle={{
            backgroundColor: 'transparent',
            alignSelf: 'flex-end',
          }}
          suggestionsListContainerStyle={{
            width: width * 0.76,
            marginLeft: -40,
            backgroundColor: themes[theme].backgroundColor,
          }}
          containerStyle={{width: '95%'}}
          showChevron={false}
        />
        <RightIcon />
      </View>
    </>
  );
};

export default CsAutocompleteSelect;
