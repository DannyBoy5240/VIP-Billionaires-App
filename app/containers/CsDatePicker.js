import React, { useState } from 'react';
import { StyleSheet, Text, TouchableOpacity, Platform, View } from 'react-native';
import DateTimePicker from '@react-native-community/datetimepicker';
import moment from 'moment';
import { DATE_STRING_FORMAT, TIME_STRING_FORMAT } from '../utils/datetime';
import { themes } from '../constants/colors';
import sharedStyles from '../views/Styles';
import { isIOS } from '../utils/deviceInfo';

const styles = StyleSheet.create({
  container: {
    flexDirection: 'row',
    alignItems: 'center',
    borderBottomWidth: StyleSheet.hairlineWidth,
    borderBottomColor: 'grey',
    marginBottom: 10,
  },
  label: {
    fontSize: 14,
    width: 80,
    paddingLeft: 8,
    paddingVertical: 4,
    ...sharedStyles.textSemibold,
  },
  input: {
    ...sharedStyles.textRegular,
    fontSize: 16,
    paddingHorizontal: isIOS ? 8 : 14,
    paddingVertical: 4,
    flexGrow: 1,
  },
});

export default function CsDatePicker({
  label,
  containerStyle,
  placeholder,
  style,
  type,
  value,
  action,
  minimumDate,
  theme,
}) {
  const [show, onShow] = useState(false);
  const [currentDate, onChangeDate] = useState(value);

  const onChange = ({ nativeEvent: { timestamp } }, date) => {
    if (Platform.OS === 'android') {
      onShow(false);
    }
    if (type === 'date') {
      console.log('date', date);
      if (!date) {
        return;
      }
      const newDate = date;
      onChangeDate(newDate);
      action({ value: moment(newDate).format(DATE_STRING_FORMAT) });
    } else {
      console.log('time', timestamp);
      if (!timestamp) {
        return;
      }
      const newDate = new Date(timestamp);
      onChangeDate(newDate);
      action({ value: moment(newDate).format(TIME_STRING_FORMAT) });
    }
  };

  // const getTimeString = date => {
  //   let hours = date.getHours();
  //   let minutes = date.getMinutes();
  //   let ampm = hours >= 12 ? 'PM' : 'AM';
  //   hours = hours % 12;
  //   hours = hours ? hours : 12; // the hour '0' should be '12'
  //   minutes = minutes < 10 ? '0' + minutes : minutes;
  //   let strTime = hours + ':' + minutes + ' ' + ampm;
  //   return strTime;
  // };

  if (Platform.OS === 'ios') {
    return (
      <View style={[styles.container, containerStyle]}>
        {label ? (
          <Text
            contentDescription={null}
            accessibilityLabel={null}
            style={[styles.label, { color: themes[theme].infoText }]}>
            {label}
          </Text>
        ) : null}
        <DateTimePicker
          testID='date-time-picker'
          style={{ width: 116 }}
          mode={type}
          is24Hour={true}
          display="default"
          minimumDate={minimumDate}
          value={currentDate ?? new Date()}
          onChange={onChange}
          textColor={'black'}
          themeVariant={'light'}
        />
      </View>
    );
  }

  let button = (
    <TouchableOpacity style={styles.input} onPress={() => onShow(!show)}>
      <Text>
        {currentDate
          ? moment(currentDate).format(
            type === 'date' ? DATE_STRING_FORMAT : TIME_STRING_FORMAT,
          )
          : placeholder}
      </Text>
    </TouchableOpacity>
  );

  const content = show ? (
    <DateTimePicker
      style={style}
      mode={type}
      display="default"
      minimumDate={minimumDate}
      value={currentDate ?? new Date()}
      onChange={onChange}
      textColor={'black'}
    />
  ) : null;

  return (
    <View style={[styles.container, containerStyle]}>
      {label ? (
        <Text
          contentDescription={null}
          accessibilityLabel={null}
          style={[styles.label, { color: themes[theme].infoText }]}>
          {label}
        </Text>
      ) : null}
      {button}
      {content}
    </View>
  );
}
