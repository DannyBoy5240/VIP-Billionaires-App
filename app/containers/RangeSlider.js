import React, { useState } from 'react'
import { StyleSheet, Text, View } from 'react-native'
import Slider from 'rn-range-slider'
import { themes } from '../constants/colors'

const THUMB_RADIUS = 12
const styles = StyleSheet.create({
  container: {
    marginBottom: 8,
    borderWidth: 1,
    borderRadius: 4,
    padding: 8,
  },
  labelText: {
    marginBottom: 8,
  },
  thumb: {
    width: THUMB_RADIUS * 2,
    height: THUMB_RADIUS * 2,
    borderRadius: THUMB_RADIUS,
    borderWidth: 2,
    borderColor: '#7f7f7f',
    backgroundColor: '#ffffff',
  },
  rail: {
    flex: 1,
    height: 4,
    borderRadius: 2,
    backgroundColor: '#7f7f7f',
  },
  railSelected: {
    height: 4,
    backgroundColor: '#4499ff',
    borderRadius: 2,
  },
  notch: {
    width: 8,
    height: 8,
    borderLeftColor: 'transparent',
    borderRightColor: 'transparent',
    borderTopColor: '#4499ff',
    borderLeftWidth: 4,
    borderRightWidth: 4,
    borderTopWidth: 8,
  },
  labelContainer: {
    alignItems: 'center',
    padding: 8,
    backgroundColor: '#4499ff',
    borderRadius: 4,
  },
  label: {
    fontSize: 16,
    color: '#fff',
  },
  content: {
    flex: 1,
    flexDirection: 'row',
    alignItems: 'center',
  },
  valueText: {
    fontSize: 12,
  },
})

export const RangeSlider = ({ style, min, max, step, initValue, onValueChanged, label, theme }) => {
  const [low, setLow] = useState(initValue.low)
  const [high, setHigh] = useState(initValue.high)

  return (
    <View
      style={[styles.container, { borderColor: themes[theme].separatorColor }]}>
      <Text style={styles.labelText}>{label}</Text>
      <View style={styles.content}>
        <Text style={[styles.valueText, { color: themes[theme].actionColor }]}>
          {low}
        </Text>
        <Slider
          style={[style, { flexGrow: 1 }]}
          min={min}
          max={max}
          step={step}
          low={low}
          high={high}
          floatingLabel
          renderThumb={() => <View style={styles.thumb} />}
          renderRail={() => <View style={styles.rail} />}
          renderRailSelected={() => <View style={styles.railSelected} />}
          renderLabel={v => (
            <View style={styles.labelContainer}>
              <Text style={styles.label}>{v}</Text>
            </View>
          )}
          renderNotch={() => <View style={styles.notch} />}
          onValueChanged={(low, high) => {
            setLow(low)
            setHigh(high)
            onValueChanged(low, high)
          }}
        />
        <Text style={[styles.valueText, { color: themes[theme].actionColor }]}>
          {high}
        </Text>
      </View>
    </View>
  )
}
