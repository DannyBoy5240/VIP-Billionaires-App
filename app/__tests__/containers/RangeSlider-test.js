import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen, fireEvent } from '@testing-library/react-native';
import { RangeSlider } from '../../containers/RangeSlider';

test('renders correctly', () => {
  const tree = renderer.create(<RangeSlider min={0} max={100} step={2} initValue={{ low: 0, high: 100 }} theme={'dark'} style={[]}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
