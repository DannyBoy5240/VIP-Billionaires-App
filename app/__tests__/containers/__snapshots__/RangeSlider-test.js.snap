// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "borderRadius": 4,
        "borderWidth": 1,
        "marginBottom": 8,
        "padding": 8,
      },
      Object {
        "borderColor": undefined,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "marginBottom": 8,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 12,
          },
          Object {
            "color": undefined,
          },
        ]
      }
    >
      0
    </Text>
    <View
      style={
        Array [
          Array [],
          Object {
            "flexGrow": 1,
          },
        ]
      }
    >
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {
              "alignItems": "flex-start",
              "left": 0,
              "position": "absolute",
              "right": 0,
            },
            Object {
              "top": -0,
            },
          ]
        }
      >
        <View
          collapsable={false}
          style={
            Object {
              "opacity": 0,
              "transform": Array [
                Object {
                  "translateX": 0,
                },
              ],
            }
          }
        >
          <View
            onLayout={[Function]}
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#4499ff",
                  "borderRadius": 4,
                  "padding": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#fff",
                    "fontSize": 16,
                  }
                }
              >
                NaN
              </Text>
            </View>
          </View>
        </View>
        <View
          collapsable={false}
          style={
            Object {
              "opacity": 0,
              "transform": Array [
                Object {
                  "translateX": 0,
                },
              ],
            }
          }
        >
          <View
            onLayout={[Function]}
          >
            <View
              style={
                Object {
                  "borderLeftColor": "transparent",
                  "borderLeftWidth": 4,
                  "borderRightColor": "transparent",
                  "borderRightWidth": 4,
                  "borderTopColor": "#4499ff",
                  "borderTopWidth": 8,
                  "height": 8,
                  "width": 8,
                }
              }
            />
          </View>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "bottom": 0,
                "flexDirection": "row",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "marginHorizontal": 0,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "#7f7f7f",
                "borderRadius": 2,
                "flex": 1,
                "height": 4,
              }
            }
          />
          <View
            collapsable={false}
            style={
              Object {
                "left": 0,
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "backgroundColor": "#4499ff",
                  "borderRadius": 2,
                  "height": 4,
                }
              }
            />
          </View>
        </View>
        <View
          collapsable={false}
          onLayout={[Function]}
          style={
            Object {
              "transform": Array [
                Object {
                  "translateX": 0,
                },
              ],
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "#ffffff",
                "borderColor": "#7f7f7f",
                "borderRadius": 12,
                "borderWidth": 2,
                "height": 24,
                "width": 24,
              }
            }
          />
        </View>
        <View
          collapsable={false}
          style={
            Object {
              "position": "absolute",
              "transform": Array [
                Object {
                  "translateX": 0,
                },
              ],
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "#ffffff",
                "borderColor": "#7f7f7f",
                "borderRadius": 12,
                "borderWidth": 2,
                "height": 24,
                "width": 24,
              }
            }
          />
        </View>
        <View
          collapsable={false}
          onMoveShouldSetResponder={[Function]}
          onMoveShouldSetResponderCapture={[Function]}
          onResponderEnd={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderReject={[Function]}
          onResponderRelease={[Function]}
          onResponderStart={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          onStartShouldSetResponderCapture={[Function]}
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 12,
          },
          Object {
            "color": undefined,
          },
        ]
      }
    >
      100
    </Text>
  </View>
</View>
`;
